--- libao-0.8.0/src/Makefile.in.orig	Mon Aug 13 07:39:23 2001
+++ libao-0.8.0/src/Makefile.in	Mon Aug 13 14:27:23 2001
@@ -94,7 +94,7 @@
 lib_LTLIBRARIES = libao.la
 
 libao_la_SOURCES = audio_out.c config.c ao_null.c ao_wav.c ao_au.c ao_raw.c ao_private.h
-libao_la_LDFLAGS = -version-info @LIB_CURRENT@:@LIB_REVISION@:@LIB_AGE@
+libao_la_LDFLAGS = -version-info @LIB_CURRENT@:@LIB_REVISION@:@LIB_AGE@ -ldl
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
 CONFIG_CLEAN_FILES = 
 LTLIBRARIES =  $(lib_LTLIBRARIES)
--- libao-0.8.0/src/Makefile.am.orig	Fri Aug  3 22:56:13 2001
+++ libao-0.8.0/src/Makefile.am	Mon Aug 13 14:27:40 2001
@@ -8,7 +8,7 @@
 lib_LTLIBRARIES = libao.la
 
 libao_la_SOURCES = audio_out.c config.c ao_null.c ao_wav.c ao_au.c ao_raw.c ao_private.h
-libao_la_LDFLAGS = -version-info @LIB_CURRENT@:@LIB_REVISION@:@LIB_AGE@
+libao_la_LDFLAGS = -version-info @LIB_CURRENT@:@LIB_REVISION@:@LIB_AGE@ -ldl
 
 
 debug:
