--- libao-0.8.2/configure.in~	Sun Jan  6 04:05:56 2002
+++ libao-0.8.2/configure.in	Sun Jan  6 04:05:56 2002
@@ -29,58 +29,6 @@
 AC_LIBTOOL_DLOPEN
 AM_PROG_LIBTOOL
 
-dnl ====================================
-dnl Set some general compile options
-dnl ====================================
-
-cflags_save="$CFLAGS"
-ldflags_save="$LDFLAGS"
-if test -z "$GCC"; then
-        case $host in
-        *-*-irix*)
-                if test -z "$CC"; then
-                        CC=cc
-                fi
-                DEBUG="-g -signed"
-                CFLAGS="-O2 -w -signed"
-                PROFILE="-p -g3 -O2 -signed" ;;
-        sparc-sun-solaris*)
-                DEBUG="-v -g"
-                CFLAGS="-xO4 -fast -w -fsimple -native -xcg92"
-                PROFILE="-v -xpg -g -xO4 -fast -native -fsimple -xcg92 -Dsuncc" ;;
-        *)
-                DEBUG="-g"
-                CFLAGS="-O"
-                PROFILE="-g -p" ;;
-        esac
-else
-
-        case $host in
-        *-*-linux*)
-                DEBUG="-g -Wall -D_REENTRANT -D__NO_MATH_INLINES -fsigned-char"
-                CFLAGS="-O20 -ffast-math -D_REENTRANT -fsigned-char"
-                PROFILE="-pg -g -O20 -ffast-math -D_REENTRANT -fsigned-char";;
-        sparc-sun-*)
-                DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char -mv8"
-                CFLAGS="-O20 -ffast-math -D__NO_MATH_INLINES -fsigned-char -mv8"
-                PROFILE="-pg -g -O20 -D__NO_MATH_INLINES -fsigned-char -mv8" ;;
-        *-darwin*)
-                SILLY_CARBON_PATH="-I/System/Library/Frameworks/Carbon.framework/Versions/A/Headers"
-                DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char $SILLY_CARBON_PATH"
-                CFLAGS="-O20 -D__NO_MATH_INLINES -fsigned-char $SILLY_CARBON_PATH"
-                PROFILE="-O20 -g -pg -D__NO_MATH_INLINES -fsigned-char $SILLY_CARBON_PATH" ;;
-        *)
-                DEBUG="-g -Wall -D__NO_MATH_INLINES -fsigned-char"
-                CFLAGS="-O20 -D__NO_MATH_INLINES -fsigned-char"
-                PROFILE="-O20 -g -pg -D__NO_MATH_INLINES -fsigned-char" ;;
-        esac
-fi
-CFLAGS="$CFLAGS $cflags_save"
-DEBUG="$DEBUG $cflags_save"
-PROFILE="$PROFILE $cflags_save"
-LDFLAGS="$LDFLAGS $ldflags_save"
-
-
 AC_SUBST(DEBUG)
 AC_SUBST(PROFILE)
 
@@ -96,17 +44,6 @@
 dnl Checks for header files
 dnl ==============================
 
-dnl --------------------------------------------------
-dnl Work around FHS stupidity
-dnl --------------------------------------------------
-
-if test "$prefix" = "/usr"; then
-    MANDIR='$(datadir)/man'
-else
-    MANDIR='$(prefix)/man'
-fi
-AC_SUBST(MANDIR)
-
 dnl ==============================
 dnl Checks for types
 dnl ==============================
@@ -85,8 +85,8 @@
 
 if test "$BUILD_ESD" = "yes"; then
   AM_PATH_ESD(0.2.8, have_esd=yes, have_esd=no)
-  AM_CONDITIONAL(HAVE_ESD,test "x$have_esd" = xyes)
 fi
+AM_CONDITIONAL(HAVE_ESD,test "x$have_esd" = xyes)
 
 
 dnl Check for OSS
@@ -104,8 +104,8 @@
 if test "$BUILD_ALSA" = "yes"; then
    AC_CHECK_LIB(asound, snd_pcm_channel_params, have_alsa=yes, have_alsa=no)
    AC_CHECK_HEADER(sys/asoundlib.h, , have_alsa=no)
-   AM_CONDITIONAL(HAVE_ALSA,test "x$have_alsa" = xyes)
 fi
+AM_CONDITIONAL(HAVE_ALSA,test "x$have_alsa" = xyes)
 
 if test "x$have_alsa" = xyes; then
 	ALSA_LIBS="-lasound"
@@ -123,8 +123,8 @@
 if test "$BUILD_ALSA09" = "yes"; then
    AC_CHECK_LIB(asound, snd_pcm_open, have_alsa09=yes, have_alsa09=no)
    AC_CHECK_HEADER(alsa/asoundlib.h, , have_alsa09=no)
-   AM_CONDITIONAL(HAVE_ALSA09,test "x$have_alsa09" = xyes)
 fi
+AM_CONDITIONAL(HAVE_ALSA09,test "x$have_alsa09" = xyes)
 
 if test "x$have_alsa09" = xyes; then
 	ALSA09_LIBS="-lasound"
@@ -159,7 +159,6 @@
 
 if test "$BUILD_ARTS" = "yes"; then
   AC_PATH_PROG(ARTSC_CONFIG, artsc-config)
-  AM_CONDITIONAL(HAVE_ARTS,test "x$ac_cv_path_ARTSC_CONFIG" != x)
 
   if test "x$ac_cv_path_ARTSC_CONFIG" != x
   then
@@ -167,6 +166,7 @@
 	ARTS_LIBS=`$ac_cv_path_ARTSC_CONFIG --libs`
   fi
 fi
+AM_CONDITIONAL(HAVE_ARTS,test "x$ac_cv_path_ARTSC_CONFIG" != x)
 
 AC_SUBST(ARTS_CFLAGS)
 AC_SUBST(ARTS_LIBS)
--- libao-0.8.2/Makefile.am~	Sun Jan  6 04:31:38 2002
+++ libao-0.8.2/Makefile.am	Sun Jan  6 04:31:38 2002
@@ -6,7 +6,6 @@
 
 m4datadir = $(datadir)/aclocal
 m4data_DATA = ao.m4
-mandir = @MANDIR@
 man_MANS = libao.conf.5
 
 EXTRA_DIST = README AUTHORS CHANGES COPYING libao.spec ao.m4 acinclude.m4 $(man_MANS)
