diff -Nur ao.org/configure.in ao/configure.in
--- ao.org/configure.in	Tue Oct 10 20:22:51 2000
+++ ao/configure.in	Sun Oct 29 14:05:41 2000
@@ -246,4 +246,13 @@
 AC_SUBST(LIBAO_FILES)
 AC_SUBST(LIBAO_LIBS)
 
-AC_OUTPUT(Makefile src/Makefile doc/Makefile include/Makefile include/ao/Makefile include/ao/os_types.h include/ao/ao_libs.inc)
+AC_OUTPUT([
+Makefile
+src/Makefile.am
+src/Makefile
+doc/Makefile
+include/Makefile
+include/ao/Makefile
+include/ao/os_types.h
+include/ao/ao_libs.inc
+])
diff -Nur ao.org/doc/Makefile.am ao/doc/Makefile.am
--- ao.org/doc/Makefile.am	Sun Sep  3 11:56:05 2000
+++ ao/doc/Makefile.am	Sun Oct 29 11:26:34 2000
@@ -2,7 +2,7 @@
 
 AUTOMAKE_OPTIONS = foreign
 
-docdir = $(prefix)/doc/$(PACKAGE)-$(VERSION)
+docdir = $(prefix)/share/doc/$(PACKAGE)-$(VERSION)
 
 doc_DATA = index.html
 
diff -Nur ao.org/src/Makefile.am ao/src/Makefile.am
--- ao.org/src/Makefile.am	Sat Sep 30 03:16:14 2000
+++ ao/src/Makefile.am	Sun Oct 29 13:38:18 2000
@@ -6,8 +6,8 @@
 
 lib_LTLIBRARIES = libao.la
 
-#libao_la_SOURCES = audio_out.c @LIBAO_FILES@
-libao_la_SOURCES = audio_out.c ao_oss.c ao_wav.c ao_null.c
+libao_la_SOURCES = audio_out.c ao_wav.c ao_null.c @LIBAO_FILES@
+#libao_la_SOURCES = audio_out.c ao_oss.c ao_wav.c ao_null.c
 libao_la_LDFLAGS = -version-info @LIB_CURRENT@:@LIB_REVISION@:@LIB_AGE@
 
 EXTRA_libao_la_SOURCES = ao_alsa.c ao_irix.c ao_oss.c ao_wav.c ao_esd.c ao_null.c ao_solaris.c audio_out.c
diff -Nur ao.org/src/Makefile.am.in ao/src/Makefile.am.in
--- ao.org/src/Makefile.am.in	Thu Jan  1 01:00:00 1970
+++ ao/src/Makefile.am.in	Sun Oct 29 13:38:18 2000
@@ -0,0 +1,20 @@
+## Process this file with automake to produce Makefile.in
+
+AUTOMAKE_OPTIONS = foreign
+
+INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/include
+
+lib_LTLIBRARIES = libao.la
+
+libao_la_SOURCES = audio_out.c ao_wav.c ao_null.c @LIBAO_FILES@
+#libao_la_SOURCES = audio_out.c ao_oss.c ao_wav.c ao_null.c
+libao_la_LDFLAGS = -version-info @LIB_CURRENT@:@LIB_REVISION@:@LIB_AGE@
+
+EXTRA_libao_la_SOURCES = ao_alsa.c ao_irix.c ao_oss.c ao_wav.c ao_esd.c ao_null.c ao_solaris.c audio_out.c
+
+
+debug:
+	$(MAKE) all CFLAGS="@DEBUG@"
+
+profile:
+	$(MAKE) all CFLAGS="@PROFILE@"
diff -Nur ao.org/src/ao_esd.c ao/src/ao_esd.c
--- ao.org/src/ao_esd.c	Sun Sep  3 11:56:05 2000
+++ ao/src/ao_esd.c	Sun Oct 29 13:29:45 2000
@@ -31,7 +31,7 @@
 #include <string.h>
 
 #include <esd.h>
-#include "audio_out.h"
+#include <ao/ao.h>
 
 typedef struct ao_esd_internal_s
 {
